### PR TITLE
NOTIF-326: Searching Empty State for Event Logs

### PR DESCRIPTION
**Pull request by Yasin-Darar**
https://user-images.githubusercontent.com/80718590/159554342-1508863e-06a3-4cb1-82ee-4832125ad133.mov

### CODE REVIEW

File: src/components/EmptyStateSearch.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import { EmptyState, EmptyStateBody, EmptyStateIcon, EmptyStateVariant, Title } from '@patternfly/react-core/dist/js/components';
```
Comment (josejulio) [inline src/components/EmptyStateSearch.tsx]: We should import from the base of the library. autocomplete messes it up sometimes.

```suggestion
import { EmptyState, EmptyStateBody, EmptyStateIcon, EmptyStateVariant, Title } from '@patternfly/react-core';
```

**Review by josejulio — CHANGES_REQUESTED** [review]
Looks good, one detail I found.

File: src/components/EmptyStateSearch.tsx (hunk 1 above)
Comment (Yasin-Darar) [reply src/components/EmptyStateSearch.tsx]: Ok, that has been changed, resolving thread.

**Review by josejulio — APPROVED** [review]
lgtm!

**Pull request merged**